app/ui/browser-modern/views/browser/decorations/searchbar.jsx

```diff
@@ -16,22 +16,33 @@ import { connect } from 'react-redux';
 
 import Style from '../../../../shared/style';
 import Search from '../../../../shared/widgets/search';
+import Btn from '../../../../shared/widgets/btn';
 
 import * as UIConstants from '../../../constants/ui';
 import * as PagesSelectors from '../../../selectors/pages';
 import * as PageActions from '../../../actions/page-actions';
 import * as PageEffects from '../../../actions/page-effects';
 
-const SEARCH_STYLE = Style.registerStyle({
+const SEARCH_BAR_STYLE = Style.registerStyle({
   position: 'fixed',
   bottom: 0,
   left: 0,
   right: 0,
   height: '25px',
   padding: '0 4px',
   zIndex: UIConstants.SEARCH_BAR_INDEX,
-  borderTop: '1px solid var(--theme-content-border-color)',
+  border: '1px solid var(--theme-content-border-color)',
   color: 'var(--theme-content-color)',
+  backgroundColor: 'var(--theme-content-background)',
+  width: '300px',
+});
+
+const CLOSE_BUTTON_STYLE = Style.registerStyle({
+  margin: '6px 3px 0px 0px',
```

Comment:
This vertical centering via margins is hacky and bound to break. Please use correct flexbox rules.